src/main/resources/data/legal-entities-with-users-retail.json

```diff
@@ -0,0 +1,220 @@
+[
+  {
+  	"category": "RETAIL",
+    "users": [
+      {
+        "externalId": "user"
+      }
+    ]
+  },
+  {
+    	"category": "RETAIL",
+    "users": [
+      {
+        "externalId": "designer"
+      }
+    ]
+  },
+  {
+    	"category": "RETAIL",
+    "users": [
+      {
+        "externalId": "manager"
+      }
+    ]
+  },
+  {
+    	"category": "RETAIL",
```

Comment:
In bb-fuel the current definition of a retail user is that the LE only contains one user. It seems you are transforming companies into retail users from this point in the file. Maybe it's better to leave them out if you only want retail users?

Reply:
Done! Thanks for the suggestion